packages/type-css-modules/.gitignore

```diff
@@ -1,5 +1,6 @@
 # compiled output
-/dist/
+# /dist/
```

Comment:
The codemod `type-css-modules` formed a special case, because this repository has a few packages that assume the compiled/published `type-css-modules` to exist.

In a consuming project, using a relative path to install `type-css-modules` (i.e. `"workspace:../../packages/type-css-modules"`) allowed me to run the `lint` script locally.

I found that, in CI, that same `lint` script would fail, because the consuming package couldn't find the compiled `type-css-modules`.

As a workaround, I decided to commit the `dist` folder. This solution is simple and allows us to run the `lint` script locally and in CI.